api/permissions.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import permissions
+
+
+class IsOwnerOrReadOnly(permissions.BasePermission):
```

Comment:
ignore this, this is in the diff because I renamed the pomodoro/ directory more approriately api/